<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="viewInputImage">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>361</width>
      <height>361</height>
     </rect>
    </property>
    <property name="text">
     <string>input image display</string>
    </property>
   </widget>
   <widget class="QLabel" name="viewOutputImage">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>361</width>
      <height>361</height>
     </rect>
    </property>
    <property name="text">
     <string>Output image display</string>
    </property>
   </widget>
   <widget class="QComboBox" name="functionList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>Choose process</string>
    </property>
    <property name="maxVisibleItems">
     <number>16</number>
    </property>
    <item>
     <property name="text">
      <string>Choose process</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Salt and pepper</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Add logo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Color space</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Compute Histogram</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Equalize Histogram</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Image Erode</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Image dilate</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Image closing</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Image opening</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Blur image</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Apply Sobel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Laplacian</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sharpen Image</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Canny</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hough Lines</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hough Circles</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Harris corners</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Draw contours</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bounding box</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Enclosing circle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FAST features</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Calibration</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Load image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="applyButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>987</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
